workflow/airflow_libero/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM python:3.6-slim
+
+ENV LANG=C.UTF-8
+
+ARG AIRFLOW_VERSION=1.9.0
+ARG AIRFLOW_HOME=/usr/local/airflow
+
+WORKDIR ${AIRFLOW_HOME}
+
+COPY Pipfile ${AIRFLOW_HOME}
+COPY Pipfile.lock ${AIRFLOW_HOME}
+COPY script/entrypoint.sh /entrypoint.sh
```

Comment:
Change to this file has just triggered a big rebuild. Might be worth moving this and the config to the end, and everything before `pipenv install` above `COPY Pipfile` (a bit of acceptable optimisation!).

Reply:
I have moved the `entrypoint.sh` and `config` `COPY`s now 👍 , but moving and splitting the `pipenv` COPY and install commands caused issues with the `WORKDIR` and it could not find the `Pipfile` 🤔 . Leaving that part for now.

Reply:
Works for me so will commit.